multimedia/imagemagick/Makefile

```diff
@@ -0,0 +1,140 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=imagemagick
+PKG_VERSION:=7.0.8
+PKG_REVISION:=49
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Val Kulkov <val.kulkov@gmail.com>
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_REVISION).tar.gz
+PKG_SOURCE_URL:=http://github.com/ImageMagick/ImageMagick/archive/$(PKG_VERSION)-$(PKG_REVISION)
```

Comment:
Any reason why official release tarball isn't used?
https://imagemagick.org/script/mirror.php --> http://ftp.acc.umu.se/mirror/imagemagick.org/ftp/ImageMagick-7.0.8-49.tar.xz etc

Reply:
Yes. As soon as a new incremental version is released, which happens about every couple of weeks with ImageMagick, the official mirrors tend to "lose" the earlier versions. When that happens, OpenWrt's robots will get 404 Not Found.

For this reason, we should stick with Github as the source repo for ImageMagick versions.